src/main/java/rlbot/cppinterop/RLBotDll.java

```diff
@@ -77,6 +78,14 @@ public static void initialize(final String interfaceDllPath) throws IOException
             System.out.println("Loading DLL from " + dllSource);
             Native.register(dllNameSansExtension);
 
+            while (!IsInitialized()) {
```

Comment:
Maybe add a maximum amount of time that we wait for?

Reply:
Good idea, done.